ci-automation/vms.sh

```diff
@@ -140,6 +140,8 @@ function _vm_build_impl() {
             COMPRESSION_FORMAT="zip"
         elif [[ "${format}" =~ ^(scaleway)$ ]];then
             COMPRESSION_FORMAT="none"
+        elif [[ "${format}" =~ ^(kubevirt)$ ]];then
+            COMPRESSION_FORMAT="gz"
```

Comment:
The qcow2 is already compressed, do we need the additional gz wrapper or can we use `none`?

Reply:
I would like to be visibile that the compression format has been applied, in case we add a raw image at some time too.
Also, it is an implementation detail that qcow2 can also support compression inside, but the naming is not implied.

Reply:
The CDI https://kubevirt.io/user-guide/operations/containerized_data_importer/ supports RAW/QCOW2 and GZ/XZ.
I ll need to add a how to on the website as well where to document that the compression format is supported.